File: Practice problem/star/star.cs
// 연습문제 짝수와 홀수  Level 1

public class Solution {
    public string solution(int num) {
        string answer = "Even";
        if(num % 2 == 1 || num % 2 == -1) {
            answer = "Odd";
        }
        return answer;
    }
}

File: Practice problem/star/star.c
// 연습문제 직사각형 별찍기  Level 1

#include <stdio.h>

int main(void) {
    int a, b;
    scanf("%d %d", &a, &b);
    for(int i = 0; i < b; i++){
        for(int j = 0; j < a; j++){ printf("*"); }
        printf("\n");
    }
    return 0;
}
